src/XMakeCommandLine/Resources/xlf/Strings.cs.xlf

```diff
@@ -327,7 +327,7 @@ Copyright (C) Microsoft Corporation. Všechna práva vyhrazena.
         <target state="translated">  /consoleloggerparameters:&lt;parametry&gt;
                      Parametry protokolovacího nástroje konzoly. (Short form: /clp)
                      Dostupné parametry:
-                        PerformanceSummary – zobrazí dobu zpracování úloh,
+                        PerformanceSummary –  zobrazí dobu zpracování úloh,
```

Comment:
Sure. I’ve made a new commit with the fix.

Best regards,

Zbyněk Sailer | Logistics IPE
T: +420-545-552-396 | Mobile: +420-731-450-569
v-zbsail@microsoft.com<mailto:v-zbsail@microsoft.com> | zsailer@moravia.com<mailto:zsailer@moravia.com> | moravia.com<http://www.moravia.com/>

From: Mihai Codoban [mailto:notifications@github.com]
Sent: Monday, July 25, 2016 6:06 PM
To: Microsoft/msbuild <msbuild@noreply.github.com>
Cc: Zbynek Sailer (Moravia IT) <v-zbsail@microsoft.com>; Author <author@noreply.github.com>
Subject: Re: [Microsoft/msbuild] LOC CHECKIN | MSBuild 20160722 | XLFs localized (#811)


In src/XMakeCommandLine/Resources/xlf/Strings.cs.xlf<https://github.com/Microsoft/msbuild/pull/811#discussion_r72092930>:

> @@ -327,7 +327,7 @@ Copyright (C) Microsoft Corporation. Všechna práva vyhrazena.

>          <target state="translated">  /consoleloggerparameters:&lt;parametry&gt;

>                       Parametry protokolovacího nástroje konzoly. (Short form: /clp)

>                       Dostupné parametry:

> -                        PerformanceSummary – zobrazí dobu zpracování úloh,

> +                        PerformanceSummary –  zobrazí dobu zpracování úloh,

Can you please remove the whitespace then?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Microsoft/msbuild/pull/811/files/ad07f0e2327bb8dd0c5d73cd17e8454aa6617dee#r72092930>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ATP2syTZtxRrUCLewtPjXAe0bcBWp_C9ks5qZN71gaJpZM4JS9ag>.